src/array_api_extra/_delegation.py

```diff
@@ -508,3 +508,192 @@ def sinc(x: Array, /, *, xp: ModuleType | None = None) -> Array:
         return xp.sinc(x)
 
     return _funcs.sinc(x, xp=xp)
+
+
+def partition(
+    a: Array,
+    kth: int,
+    /,
+    axis: int | None = -1,
+    *,
+    xp: ModuleType | None = None,
```

Comment:
Wouldn't it make sense to expose a `stable` argument to `xp.partition` and `xp.argpartition`? The default value `stable` in those methods could then be made to match the default value in `xp.sort` and `xp.argsort` for consistency.

Reply:
If `xpx.argpartition(data, kth, stable=True)`  is not supported by the underlying namespace, then the call would be delegated to the `xp.argsort`-based slow but stable fallback.